safe_transaction_service/tokens/views.py

```diff
@@ -37,11 +35,7 @@ def get(self, request, *args, **kwargs):
                 },
             )
 
-        try:
-            return super().get(request, *args, **kwargs)
-        except Http404 as exc:  # Try to get info about the token
-            get_token_info_from_blockchain_task.delay(address)
-            raise exc
+        return super().get(request, *args, **kwargs)
```

Comment:
Maybe we can rename `Cache 1 hour, this should never change` to `Cache 1 hour, this does not change often`

Reply:
👍🏻 changed in [50506cc](https://github.com/safe-global/safe-transaction-service/pull/1162/commits/50506cc3a626cd0bd6eb31ecf182bce222aba16c)